qa/paqa_devs.c

```diff
@@ -317,139 +674,132 @@ static void TestDevices( int mode, int allDevices )
         }
         if( maxChannels > MAX_TEST_CHANNELS )
             maxChannels = MAX_TEST_CHANNELS;
+        if (maxChannels == 0) continue;  // skip this device, wrong direction
 
         if (!allDevices && !isDefault) continue; // skip this device
 
+        printf("\n===========================================================\n");
+        printf("            Device = %s\n", pdi->name );
+        printf("===========================================================\n");
         for( jc=1; jc<=maxChannels; jc++ )
         {
-            printf("\n===========================================================\n");
-            printf("            Device = %s\n", pdi->name );
-            printf("===========================================================\n");
+            printf("\n---------------------- NumChannels = %d ------------\n", jc );
+            if (mode == MODE_INPUT) {
+                parameters.numInputChannels = jc;
+            } else {
+                parameters.numOutputChannels = jc;
+            }
             /* Try each standard sample rate. */
-            for( i=0; standardSampleRates[i] > 0; i++ )
+            for( i=0; kStandardSampleRates[i] > 0; i++ )
```

Comment:
I am not comfortable with letting this old Macintosh/Android naming convention into the tree. Let's stick to our style guide.

Reply:
I find the Hungarian notation to be very helpful and improves readability. But that is a discussion for another day. Done.